lib/galaxy/tools/parameters/basic.py

```diff
@@ -163,9 +163,6 @@ def value_to_basic( self, value, app ):
         return self.to_string( value, app )
 
     def value_from_basic( self, value, app, ignore_errors=False ):
-        # HACK: Some things don't deal with unicode well, psycopg problem?
```

Comment:
@mvdbeek Can you comment on this? Can you be confident this hack is no longer needed?